src/Redistribution.sol

```diff
@@ -482,6 +486,21 @@ contract Redistribution is AccessControl, Pausable {
         require(false, "no matching commit or hash");
     }
 
+    /**
+     * @notice Returns minimum depth reveal has to have to participate in this round
+     */
+    function currentMinimumDepth() public view returns (uint8) {
+        // We are checking value in reveal phase, as the currentCommitRound is set to the current round
+        // but the currentClaimRound is still set to the last time claim was made
+        // We add 1 to ensure that for the next round the minimum depth is the same as last winner depth
+
+        uint8 skippedRounds = uint8(currentCommitRound - currentClaimRound) + 1;
```

Comment:
IMHO we can accept that after 256 missing rounds the last winner.depth is counting down again (roughly 54 days)

Reply:
I don't understand what was said here. :) Can you please try to explain again?

Reply:
just a note because of the possible overflow, it does not require any action. after 256 rounds the substraction will result in 0 because of the uint8 casting

Reply:
but 256 missing rounds unlikely, hopefully never happen

Reply:
It would wrap around, meaning something like if there is 300 in difference it would be 44 as result.
So I am suggesting this as solution to make some upperLimit that can be reached

```
        uint256 difference = currentCommitRound - currentClaimRound;
        uint8 skippedRounds = uint8(difference > 254 ? 254 : difference) + 1;
```